core/capabilities/targets/write_target.go

```diff
@@ -56,15 +56,13 @@ func NewWriteTarget(lggr logger.Logger, id string, cr commontypes.ContractReader
 		"Write target.",
 	)
 
-	logger := lggr.Named("WriteTarget")
```

Comment:
why did the methods on the instance disappear? the new convention is less intuitive to me.

Reply:
The common package logger interface captures only the simple methods implemented by zap. A Sugared logger can still be constructed from any instance to provide these operations as methods, but making one is messier than just calling the helper when there is just one thing to do.